### ISSUE TITLE
Can not add custom stamp from a file if the file located within folder or path named (or part of it's name) with NON-Latin characters

### ISSUE THREAD

**Issue by Nokia808** [flathub/org.kde.okular#67]
Hi. I think this is a bug related to permission of this flatpak package. 

In review, there is annotation option "Stamp". "Stamp" option in lowest of it's suboption to use custom stamp from a file located on hard desk. I used it & select a .png file which shown to me, & it appeared to me within stamp area. But when I click "Save" option, then closed Okular, then relaunch it again, I saw stamp area empty. When I investigated, I saw the stamp dialog is still existing, but without content ! I retry again & again with same negative result !

**Comment by Nokia808** [comment]
I test issue on version 1.10 & it is fixed. I will close this issue. But was it a flatpak package bug or okular bug ?

**Comment by Nokia808** [comment]
I re-test it on the current version 1.10.3 & the bug return again !!

I will re-open this bug.

**Comment by Nokia808** [comment]
@tsdgeos 
Hi. Finally I discovered the reality of this issue !! The problem lie in the following problem: adding stamp tool currently not supporting non Latin (or at least Arabic) character in the name of directory path for image ! When I opened this issue, I was tested it with image stored in a folder that I use Arabic name for it (so it did not work & I opened this issue). And when I closed this issue, I was tested it on image stored in other folder named in English name (so it worked correctly). When I re-opened this issue, I was tested it on image located within folder named using Arabic characters (so did not work & I re-opened the issue).

Today, I test it on image located in folder named in English so it worked. But I re-test it with other image located in other folder named in Arabic & it did not work. At this stage I saw the stamp tool show the Arabic part of directory path for image as "??????". This bring my attention & I tried to changed the name of the folder from Arabic to English & then retried to add the image & it worked !

I will keep this issue opened after changing it's title to indicate the real nature of it.
